go_node_engine/cmd/logs.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"fmt"
+	"go_node_engine/logger"
+	"io"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(logsCmd)
+}
+
+var (
+	logsCmd = &cobra.Command{
+		Use:   "logs",
+		Short: "tail check node engine logs",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return logsNodeEngine()
+		},
+	}
+)
+
+func logsNodeEngine() error {
+	// Open the log file
```

Comment:
nit: I think we can remove this comment because it is clear what the line below does